prov/verbs/src/fi_verbs.h

```diff
@@ -334,29 +337,35 @@ typedef int(*fi_ibv_mr_reg_cb)(struct fi_ibv_domain *domain, void *buf,
 			       struct fi_ibv_mem_desc *md);
 typedef int(*fi_ibv_mr_dereg_cb)(struct fi_ibv_mem_desc *md);
 
+void fi_ibv_mem_notifier_free_hook(void *ptr, const void *caller);
+void *fi_ibv_mem_notifier_realloc_hook(void *ptr, size_t size, const void *caller);
+
+struct fi_ibv_mem_notifier;
+
 struct fi_ibv_domain {
-	struct util_domain	util_domain;
-	struct ibv_context	*verbs;
-	struct ibv_pd		*pd;
+	struct util_domain		util_domain;
+	struct ibv_context		*verbs;
+	struct ibv_pd			*pd;
 	/*
 	 * TODO: Currently, only 1 rdm EP can be created per rdm domain!
 	 *	 CM logic should be separated from EP,
 	 *	 excluding naming/addressing
 	 */
-	enum fi_ep_type		ep_type;
-	struct fi_ibv_rdm_cm	*rdm_cm;
-	struct slist		ep_list;
-	struct fi_info		*info;
+	enum fi_ep_type			ep_type;
+	struct fi_ibv_rdm_cm		*rdm_cm;
+	struct slist			ep_list;
+	struct fi_info			*info;
 	/* This EQ is utilized by verbs/RDM and verbs/DGRAM */
-	struct fi_ibv_eq	*eq;
-	uint64_t		eq_flags;
+	struct fi_ibv_eq		*eq;
+	uint64_t			eq_flags;
```

Comment:
Please please avoid fixing unrelated white space issues in the future. It makes cherry-picking difficult. :)

Reply:
Sure, will take it into account in the future.